### ISSUE TITLE
cmd/storagenode: internal upload error in piecestore

### ISSUE THREAD

**Issue by dsnet** [storj/storj#7098]
I'm seeing a large number of error logs of the form:
```
ERROR	piecestore	upload internal error	{"Process": "storagenode", "error": "manager closed: unexpected EOF", "errorVerbose": "manager closed: unexpected EOF\n\tgithub.com/jtolio/noiseconn.(*Conn).readMsg:225\n\tgithub.com/jtolio/noiseconn.(*Conn).Read:171\n\tstorj.io/drpc/drpcwire.(*Reader).read:68\n\tstorj.io/drpc/drpcwire.(*Reader).ReadPacketUsing:113\n\tstorj.io/drpc/drpcmanager.(*Manager).manageReader:230"}
```
An unexpected EOF may well be a normal failure case, but perhaps being misclassified as an "internal error" which feels more serious?

All nodes with this error are running v1.109.2.

\cc @jtolio (since it uses your `noiseconn` package 😄

**Comment by nerdatwork** [comment]
https://github.com/storj/storj/issues/7074

**Comment by mniewrzal** [comment]
Should be fixed in v1.113.2
